lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -37,7 +39,9 @@
                                 model: userAccountModel,
                                 title: gettext('Full Name'),
                                 valueAttribute: 'name',
-                                helpMessage: gettext('The name that appears on your edX certificates. Other learners never see your full name.')
+                                helpMessage: interpolate_text(
+                                    gettext('The name that appears on your {platform_name} certificates. Other learners never see your full name.'), {platform_name: platformName}
```

Comment:
We probably don't need the platform name here, so if we want to take it out, I think that's fine.